<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-GLJB1TD" failures="1" tests="4" name="Selenium Course Suite tests" time="42.39" errors="0" timestamp="2021-04-26T00:04:11 EEST">
  <testcase classname="com.finance.yahoo.test.testcases.YahooSignUp" name="executeTest" time="16.062"/>
  <testcase classname="com.finance.yahoo.test.testcases.CheckStatistics" name="executeTest" time="19.014">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [0.82 (0.62%)] but found [0.82 (0.61%)]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [0.82 (0.62%)] but found [0.82 (0.61%)]
at com.finance.yahoo.test.testcases.CheckStatistics.executeTest(CheckStatistics.java:31)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- executeTest -->
  <testcase classname="com.finance.yahoo.test.testcases.YahooSignUp" name="executeTest" time="14.659"/>
  <testcase classname="com.finance.yahoo.test.testcases.CheckStatistics" name="executeTest" time="15.886"/>
</testsuite> <!-- Selenium Course Suite tests -->
